Jenkinsfile

```diff
@@ -22,6 +22,7 @@ pipeline {
         DOCKER_GRAPHDRIVER  = 'overlay2'
         APT_MIRROR          = 'cdn-fastly.deb.debian.org'
         CHECK_CONFIG_COMMIT = '78405559cfe5987174aa2cb6463b9b2c1b917255'
+        TESTDEBUG           = '1'
```

Comment:
Is this temporary?

Reply:
It gives a bit more information what's running; was in doubt to remove the commit entirely, or set it to `0`; wdyt?